basehub/values.yaml.j2

```diff
@@ -12,20 +10,22 @@ jupyterhub:
     singleuser:
         image:
             name: ghcr.io/aiidalab/qe
-            tag: pr-781
+            tag: edge
```

Comment:
Try `pr-795` with hq integrated?
In my plan, that image is ready for test once the hq support PR is merged. But seems @giovannipizzi want to wait for lazy loading and probably even better after use python 3.13.